### ISSUE TITLE
Callout icons should have aria-hidden="true" to indicate that the SVG is decorative in nature

### ISSUE THREAD

**Issue by styu** [palantir/blueprint#4793]
Images that are purely decorative in nature and/or have adjacent text that are explanatory already can be ignored by assistive technologies: https://www.w3.org/WAI/tutorials/images/decorative/

For callouts, the icons next to the title (which is the informative part of the component) are more than likely purely decorative, so I'd imagine it would be safe to mark them as `aria-hidden="true"` by default

#### Environment

- __Package version(s)__: 3.46.0
- __Operating System__: MacOS
- __Browser name and version__: Chrome

#### Screenshot

![image](https://user-images.githubusercontent.com/1143755/124336179-03670980-db6b-11eb-9751-499ab7a5f840.png)

**Comment by invliD** [comment]
I'm wondering whether we should hold that for all icons. Most icons are named based on what they visually represent, not what they semantically mean, so having a screenreader read the icon name is probably not helpful. In most contexts (e.g. buttons, menu items, etc.) there should be a more semantic version in text next to it anyway. For icon-only use-cases, I think it makes sense to put the burden of accessibility on the consumer, as BP can never know what the intention of an icon usage may be.

**Comment by styu** [comment]
For an icon only use case, I believe we'd want to add `aria-label` and don't want `aria-hidden`, else the label won't get read by the screen reader, if I understand correctly. I think it'd be difficult for Blueprint to definitely know what are icon-only use cases in all scenarios (e.g. I wouldn't put `aria-hidden` in the `<Icon>` component by default), but where we know the icons are decorative, I feel like it makes sense to pre-emptively add `aria-hidden` (hence this issue)
